components/admin_console/admin_definition.jsx

```diff
@@ -4349,6 +4349,32 @@ const AdminDefinition = {
                         help_text_default: 'Enables channel sidebar organization options in **Account Settings > Sidebar > Channel grouping and sorting** including options for grouping unread channels, sorting channels by most recent post and combining all channel types into a single list.',
                         help_text_markdown: true,
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_DROPDOWN,
+                        key: 'ServiceSettings.ExperimentalChannelSidebarOrganization',
+                        label: t('admin.experimental.experimentalChannelSidebarOrganization.title'),
+                        label_default: 'Sidebar Organization - TODO DEVIN:',
```

Comment:
Is there a ticket to go update this text? If not, could you create one?

Reply:
I figured I would get the text before this PR got submitted, but we can break it up.

Reply:
https://mattermost.atlassian.net/browse/MM-21468